packages/suite/src/components/suite/WordInput.tsx

```diff
@@ -1,11 +1,9 @@
-import { Children, ReactElement, memo, useEffect, useRef } from 'react';
+import { Children, ReactElement, memo, useEffect, useRef, useState } from 'react';
 import { MenuListProps, SelectInstance, createFilter } from 'react-select';
 import { useKeyPress } from 'react-use';
 import { FixedSizeList as List } from 'react-window';
```

Comment:
Do we need this? Looks like we use `react-window` just because of this. And we have only 2048 items here AFAIK. Couldn't we delete it? 🤔